metadata-db-app/project.clj

```diff
@@ -12,7 +12,7 @@
   :dependencies [[cheshire "5.8.1"]
                  [clj-time "0.15.1"]
                  [com.fasterxml.jackson.core/jackson-core "2.9.8"]
```

Comment:
We should try to keep versions consistent across modules. Let's update jackson-core to match the 2.10.5 version.

Reply:
Still trying to get a stable version/build, but this will be updated to be consistent with whatever jackson version is chosen.